### ISSUE TITLE
Feature: Notification before event ending

### ISSUE THREAD

**Issue by yarsanich** [leits/MeetingBar#246]
I was wondering how it would be nice to have a reminder that says "5 minutes left from [Event name]". I think it may prevent us from extended meetings.

Notification could be the same as before the meeting. And it would be nice to have a configurable time range - when notification will be delivered (E.G 1 min, 5 min, 10 min before meeting end)

My use case: time blocking technique. I have time blocks in my schedule for deep work and I would like to get notified before the time block ends. It's kinda not related to meetings, but I think it may be useful for others too.

**Comment by jgoldhammer** [comment]
Thanks @yarsanich - nice feature request. Still thinking about this. 

Even more I could imagine that we could give organisers of the event the possibility to extend the meeting for 5, 10 or 15 minutes... WDYT @leits  and @yarsanich ?

**Comment by yarsanich** [comment]
@jgoldhammer extending event's would be nice addition to it I agree.

**Comment by jgoldhammer** [comment]
I am still working on having notifications for ending events. 

Extending in the calendar should technically work.  Currently we have a readonly access.
Do we want that to include that into Meetingbar, @leits ?

**Comment by Tom-Potanski** [comment]
Agreed, very important one. Even no need for an option to auto extend the meeting.

Point is - meetings that take longer than planned are a huge time waste. Some kind of a notification (either system notif. or a full screen - depending on user's decision/configuration) would be absolutely GREAT improvement and a massive productivity gain.

@leits

**Comment by davidneedham** [comment]
The beta feature works for me.
